feature-libs/commerce-quotes/.release-it.json

```diff
@@ -0,0 +1,36 @@
+{
+  "git": {
+    "requireCleanWorkingDir": true,
+    "requireUpstream": false,
+    "tagName": "commerce-quotes-${version}",
+    "commitMessage": "Bumping commerce quotes version to ${version}",
+    "tagAnnotation": "Bumping commerce quotes version to ${version}"
+  },
+  "npm": {
+    "publishPath": "./../../dist/commerce-quotes"
+  },
+  "hooks": {
+    "after:version:bump": "cd ../.. && yarn build:commerce-quotes"
+  },
+  "github": {
+    "release": true,
+    "assets": ["../../docs.tar.gz", "../../docs.zip"],
+    "releaseName": "@spartacus/commerce-quotes@${version}",
+    "releaseNotes": "ts-node ../../scripts/changelog.ts --verbose --lib commerce-quotes --to commerce-quotes-${version}"
+  },
+  "plugins": {
+    "../../scripts/release-it/bumper.js": {
+      "out": [
+        {
+          "file": "package.json",
+          "path": [
+            "peerDependencies.@spartacus/core",
+            "peerDependencies.@spartacus/storefront",
+            "peerDependencies.@spartacus/schematics",
+            "peerDependencies.@spartacus/styles"
```

Comment:
I think for now I would leave only schematics in peers. We're not certain if other libs will be required